realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/RealmFunctionsTest.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm
+
+import androidx.test.platform.app.InstrumentationRegistry
+import io.realm.internal.util.BsonConverter
+import org.bson.*
+import org.junit.Test
+import java.util.stream.Collectors
+import kotlin.test.assertEquals
+
+class RealmFunctionsTest {
+
+    @Test
+    operator fun invoke() {
+        Realm.init(InstrumentationRegistry.getInstrumentation().targetContext)
+
+        val functions = RealmFunctions()
+        val i32 = 42
```

Comment:
In other tests of this kind we normally either create a custom enum or switch on an existing one. That way it becomes really clear if you are missing tests for a specific type. In this case there is already a `BsonType` enum, right?

Reply:
Yep. You commented earlier on another PR, and just before pushing I got the point :)